changelog.md

```diff
@@ -0,0 +1,22 @@
+**Beta features**
```

Comment:
All the comments below, can be done in main readme.md for Beta description as well.
To discuss:
1. Do we need to call it `beta` or `beta preview`?
2. Do we need to keep in changelog the same list as in readme.md, or only the real changes? Moreover in changes we can have extended list with different changes, but in readme.md just most important features supported for now.